package.json

```diff
@@ -119,6 +119,7 @@
     "nunjucks": "^3.2.3",
     "passport": "^0.4.1",
     "passport-oauth2": "^1.5.0",
+    "qs": "^6.7.0",
```

Comment:
Does `qs` give us something that doesn't come with the Node `Query string` package (already part of `package.lock`)?

I've just tried it locally, and using `QueryString.stringify` seems to produce the same result - maybe we don't need to worry about importing a new package for this

Reply:
good point - i wasn't aware node had this built in. differences are there but unimportant for us:

https://stackoverflow.com/questions/29136374/what-the-difference-between-qs-and-querystring

we already have this lib pulled in as a transitive dep. so which one we use is here nor there. based on the SO answer above, since qs provides more features, and we already have it anyway, i'm happy with using it here.

Reply:
Just copying this from our slack conv: I'd rather we used Querystring as it's a dependency we don't have to worry too much about updating on its own, as it's part of Node, and we haven't needed the additional things that `qs` provides up til now.

But I'll leave it up to you - one thing, if you do decide to stick with `qs`, would you mind updating the commit message to add a short description as to why we're using it?